tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -149,6 +149,7 @@ def test_universal_autolog_calls_pyspark_immediately():
 @pytest.mark.parametrize("config", [{"disable": False}, {"disable": True}])
 def test_universal_autolog_attaches_pyspark_import_hook_if_pyspark_isnt_installed(config):
     with mock.patch("mlflow.spark.autolog", wraps=mlflow.spark.autolog) as autolog_mock:
+        autolog_mock.integration_name = "pyspark"
```

Comment:
Fix for this error:

https://github.com/mlflow/mlflow/runs/1891435825#step:5:1109

```
E           KeyError: <MagicMock name='autolog.integration_name' id='140188961525656'>
```